src/main/jni/src/genomicsdb_GenomicsDBImporter.cc

```diff
@@ -33,13 +33,28 @@
 #define GET_GENOMICSDB_IMPORTER_FROM_HANDLE(X) \
     (reinterpret_cast<GenomicsDBImporter*>(static_cast<std::uintptr_t>(X)))
 
+void handleJNIImporterException(JNIEnv *env, std::exception& exception) {
+  std::string msg = std::string("GenomicsDB JNI Importer Error: ") + exception.what();
+  jclass io_exception_class = env->FindClass("java/io/IOException");
+  if (io_exception_class) {
+    jboolean flag = env->ExceptionCheck();
+    if (flag) {
+      env->ExceptionClear();
+    }
+    env->ThrowNew(io_exception_class, msg.c_str());
+  } else {
+    throw std::runtime_error(msg);
+  }
+}
+
 JNIEXPORT jint JNICALL
 Java_org_genomicsdb_importer_GenomicsDBImporterJni_jniGenomicsDBImporter(
   JNIEnv* env,
   jobject obj,
   jstring loader_configuration_file,
   jint rank) {
 
+  try {
   //Java string to char*
```

Comment:
nit:  could you fix the indents here and below where the try-catch have been added?